plugins/ODbgRegisterView/DialogEditFPU.cpp

```diff
@@ -38,7 +38,7 @@ long double readFloat(const QString &strInput, bool &ok) {
 	ok = false;
 	const QString str(strInput.toLower().trimmed());
 
-	if(const auto value = util::fullStringToFloat<long double>(str.toStdString())) {
+	if (const auto value = util::fullStringToFloat<long double>(str.toStdString())) {
```

Comment:
I think this should be put into the Style Guide. Currently not only doesn't it tell to put space after `if`, but its examples also don't follow this rule.

Reply:
Agreed, it is however enforced by clang-format :+1: . It will be added to the style guide.